peer/brontide.go

```diff
@@ -2538,13 +2590,59 @@ func (p *Brontide) reenableActiveChannels() {
 	// channel is already active, the update won't be sent.
 	for _, chanPoint := range activePublicChans {
 		err := p.cfg.ChanStatusMgr.RequestEnable(chanPoint, false)
```

Comment:
Can `ChanStatusMgr.RequestEnable(chanPoint, false)` be called concurrently from multiple go routines?

If yes, then a re-work could be considered where the extra attempts to enable a channel are done in a go routine, possibly attempting multiple times within a wait duration, with attempts occurring every `retryEnableInterval`. This way the attempts would be more persistent. Of course this would be more cpu intensive, only to be considered as a work-around if a single `retryEnableInterval` is not long enough to wait for the link activation line.

Reply:
`reenableActiveChannels` will only be called on start once so nope.